lib/ansible/modules/windows/win_say.ps1

```diff
@@ -36,7 +36,7 @@ $result = @{
 $speed = 0
 $words = $null
 
-if ($speech_speed -ne $null) {
+if ($speech_speed) {
```

Comment:
No idea if `$speech_speed` can be `0` but if an int is 0 then this won't be true. I would keep this as `if ($speech_speed -ne $null)`